guides/user/kubernetes-v2/parameterize-manifests/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: single
+title:  "Parameterize Kubernetes Manifests"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.6" %}
+
+{% include toc %}
+
+It is possible to have Spinnaker inject context from the currently executing
+pipeline into your manifests as they are deployed, whether they are deployed
+[statically](/guides/user/kubernetes-v2/deploy-manifest/#specifying-manifests-statically)
+or
+[dynamically](/guides/user/kubernetes-v2/deploy-manifest/#specifying-manifests-dynamically).
+
+This can be applied to a wide-range of use-cases, but we will focus on using a
+pipeline parameter to specify the target namespace.
+
+## Configure your Pipeline Parameters
+
+First we will register a pipeline parameter in the "configuration" tab of the
+pipeline editor (only the __Name__ is required):
+
+{% include
+   figure
+   image_path="./parameter.png"
+%}
+
+> See more details on how to provide parameters to piplines programatically in 
+> the [wehbooks](/guides/user/triggers/webhooks) page.
+
+## Configure your Manifest
+
+We want the user-supplied parameter to specify the manifest's namespace. Edit
```

Comment:
Maybe make it clearer that this is an example.

"In this scenario, we're using a parameter to specficy the manifest's namespace."

Or even "We want _this_ user-supplied parameter to..."

Reply:
(Consider the case where the careless reader skips the intro sentence of this article and goes right for the meat of the instructions.)